server/utils/request.ts

```diff
@@ -0,0 +1,40 @@
+import { RequestInterface } from '../models/requestModel'
+import { ServerResponseInterface } from '../graphql/serverResponse'
+import { UserInputError } from 'apollo-server-errors'
+
+import { updateRequestTypeHelper } from './requestType'
+
+const filterOpenRequests = (requests: Array<RequestInterface> ) => {
+  return requests.filter(request => request.fulfilled === false && request.deleted === false)
+}
+
+const filterFulfilledRequests = (requests: Array<RequestInterface> ) => {
+  return requests.filter(request => request.fulfilled === false && request.deleted === false)
```

Comment:
Also, I don't think it should matter if request.deleted is true or false if you are filtering solely by fulfilled.

Reply:
See this slack thread: https://uwblueprint.slack.com/archives/C01RCAMC17F/p1616119306013300